src/modules/DataStore/StatisticStore.ts

```diff
@@ -114,108 +117,122 @@ export default class Statistics implements Saveable {
         'shinyPokemonDefeated',
         'shinyPokemonEncountered',
         'shinyPokemonHatched',
-    ]
+    ];
 
     constructor() {
-        for (const prop of this.observables) {
-            this[prop] = ko.observable(0).extend({ numeric: 0 });
```

Comment:
Is there a reason we are removing all the extensions?

As with just `<number>` we can still end up with decimal values which has happened in the past.

Reply:
I had looked through everything using these values and didn't see anything that could generate decimal numbers. Knockout extenders don't play nicely with TS (a lot of errors and complexity around `extend does not accept object property "numeric"`), and I was hoping it was some dead code, since `numeric` also had support for the level of decimal precision but everything used it to just strip all decimals. I'll see if I can find a clean way of restricting the observable and push a fix :+1: